docs/sphinx/manual/creating_hpx_projects.rst

```diff
@@ -297,6 +297,29 @@ variable to point to the installation of |hpx|.
    in your targets. The |hpx| include directories are available with the
    ``HPX_INCLUDE_DIRS`` CMake variable.
 
+.. _hpxcxx_documentation:
+
+Building |hpx| components and applications
```

Comment:
I think calling this `Using the |hpx| compiler wrapper ``hpxcxx`` ` might make more sense (since this isn't the only way to compile HPX components and applications.

Reply:
Got it. I'll make the changes.